webpack.config.js

```diff
@@ -13,14 +13,25 @@ module.exports = {
     rules: [
       {
         test: /\.jsx?$/,
-        exclude: /(node_modules|bower_components)/,
+        // exclude: /(node_modules|bower_components)/,
+        // according to docs, they suggest to choose INCLUDE over EXCLUDE
+        include: path.join(__dirname, './client'),
         use: {
           loader: 'babel-loader',
           options: {
             presets: ['@babel/preset-env', '@babel/preset-react']
           }
         },
+      },
+      {
+        // Preprocess our own .css files
+        // This is the place to add your own loaders (e.g. sass/less etc.)
+        // for a list of loaders, see https://webpack.js.org/loaders/#styling
+        test: /\.css$/,
```

Comment:
Are you planning on keeping your css in a styles.css file or using JSS / styled-components / etc. ?